src/hooks/use-sync-states-progress-info.ts

```diff
@@ -130,12 +137,59 @@ export function useSyncStatesProgressInfo() {
 		[]
 	);
 
+	const getBackupStatusWithProgress = useCallback(
+		(
+			hasBackupCompleted: boolean,
```

Comment:
Couldn't we intuit this value inside `getBackupStatusWithProgress` by checking if `response.status === 'finished'`?